packages/core/src/errors/RequestError/index.ts

```diff
@@ -57,4 +66,25 @@ export default class RequestError extends Error {
 
     return conditional(this.data instanceof ZodError && formatZodError(this.data).join('\n'));
   }
+
+  /**
+   * Parse the error message with i18n context
```

Comment:
this is inaccurate. should be something like:
```suggestion
   * Parse the error into response body. An i18n context is required.
```